.github/ISSUE_TEMPLATE.md

```diff
@@ -4,71 +4,56 @@ labels: 'tech-debt'
 
 ---
 
-<!-- IF you don't have any info, just erase for the template -->
-
-# Tech Debt Title
+<!-- If you don't have any info, just erase the template -->
 
 ## Summary
 
-> TL;DR: A simple summary about the tech debt
-
-**Feature related:**
+TL;DR: A simple summary about the tech debt.
 
-**Age:** { age:legacy | age:new-tech-debt-introduced }
-
-**Present since:** 201X-XX-XX
+**Related feature:** Is it related to what major feature (e.g. visits, crm, contracts)?
 
 <!--
-  Age labels:
+Those below are going to be transformed into labels:
 
-  - LEGACY: found in the codebase and being document right now.
+Age labels:
 
-  - NEW_TECH_DEBT_INTRODUCED: we consciously added a new tech debt in our code base right now and we're document it.
--->
+  - LEGACY: found in the codebase and being documented right now.
 
-**Estimated cost:** { estimatedcost:simple | estimatedcost:complex | estimatedcost:investigation_needed }
-<!-- going to transform also in a label -->
+  - NEW_TECH_DEBT_INTRODUCED: we intentionally added a new tech debt in our codebase right now and we're documenting it.
 
-<!--
-  Estimated cost labels:
+Estimated cost labels:
 
-  - SIMPLE: It's simple to identify the solution and clearly simple to solve and test.
+  - SIMPLE: It's simple to identify the solution, fix it and test it.
 
-  - COMPLEX: It's not so simple to understand and fix it. it also may require big refactorings.
+  - COMPLEX: It's not so simple to understand it and fix it. It may also require big refactoring.
 
-  - INVESTIGATION_NEEDED: There's a lot of unknowns related that it's impossible to say until it's prioritized, invetigated and have proposed solutions.
+  - INVESTIGATION_NEEDED: There's a lot of unknowns, so that it's impossible to say until it's prioritized, investigated
+                          and have proposed solutions.
 -->
 
-**Type:** { type:documentation | type:coding | type:testing }
-
-### Description :clipboard:
-
-A clear and concise description of what the tech debt is and the reason of being created
+**Age:** { age:legacy | age:new-tech-debt-introduced }
 
-### Impact :bomb:
+**Present since:** 20XX-XX-XX
 
-Description of the current or possible impact of this tech debt.
+**Estimated cost:** { estimatedcost:simple | estimatedcost:complex | estimatedcost:investigation_needed }
 
-<!-- estimated -->
-**Critical in**: { N MONTHS | N YEARS | UNKOWN }
+**Type:** { type:documentation | type:coding | type:testing }
 
-or
+## Description :clipboard:
 
-**Critical when**
+Give a clear and concise description of what the tech debt is.
 
-### Solution Hints :shipit:
+## Impact :bomb:
 
-<!--
-Here you should add only hints, context, and opinions
-but let the solution proposal in a PR for ADR.
--->
+Include a description of the current or possible impact of this tech debt and when it's estimated to become critical.
```

Comment:
Maybe we can keep the new hint texts as comments, just like the label explanation at the top, so people won't have to delete it when filling in the sections

```suggestion
<!--
Include a description of the current or possible impact of this tech debt
and when it's estimated to become critical.
-->
```